### DISCUSSION TITLE
I have two questions.

### DISCUSSION THREAD

**Discussion by cocolalayh** [petercorke/robotics-toolbox-python#303] [Q&A]
I'm sorry, I'm new to this field
How can I use the toolbox to create my own robot?
How do I use the toolbox for arm trajectory planning?

**Comment by petercorke** [comment]
That's a big question.  You need to be familiar with underlying concepts, the approach in Robotics, Vision & Control book is SIMILAR to this.  Have a look at the examples, included with the toolbox, as well.
